.gitignore

```diff
@@ -2,6 +2,7 @@
 #
 #
 
+dist
```

Comment:
Wait I think we actually do want to push some dist for ts so that you can get non relative js imports. @Metachaser24 added this I believe

Reply:
@isidroamv made those changes in the end

Reply:
`dist` was removed the repo - `yarn build` will run before `yarn publish` and dist is set to be part of npm package.